tests/OperationResponseTest.php

```diff
@@ -116,6 +124,10 @@ public function testWithOptions()
         $this->assertNull($op->getResult());
         $this->assertNull($op->getError());
         $this->assertNull($op->getMetadata());
+        $this->assertEquals([
+            'operationReturnType' => '\google\rpc\Status',
+            'metadataReturnType' => '\google\rpc\Status',
```

Comment:
Having both of these set to the same thing means your test might overlook assignment mixups in the implementation; try to have your unit test inputs use distinct values if possible.